src/wazuh_db/wdb_agents.c

```diff
@@ -88,24 +88,30 @@ cJSON* wdb_agents_insert_vuln_cves(wdb_t *wdb,
                                    const char* reference,
                                    const char* type,
                                    const char* status,
-                                   bool check_pkg_existence) {
+                                   bool check_pkg_existence,
+                                   const char* severity,
+                                   double cvss2_score,
+                                   double cvss3_score) {
 
+    bool insert = FALSE;
     cJSON* result = cJSON_CreateObject();
     if (!result) {
         return NULL;
     }
 
     if (wdb_agents_find_cve(wdb, cve, reference)) {
         cJSON_AddStringToObject(result, "action", "UPDATE");
+        insert = FALSE;
     }
     else {
         cJSON_AddStringToObject(result, "action", "INSERT");
+        insert = TRUE;
     }
 
     if (check_pkg_existence && !wdb_agents_find_package(wdb, reference)) {
         cJSON_AddStringToObject(result, "status", "PKG_NOT_FOUND");
     }
-    else {
+    else if (insert){
```

Comment:
Would be possible that we would also need to update the severity, cvss2, and cvss3. Not only insert them?

Reply:
When we insert a CVE, the severity/score and the rest of the related information can't be changed (unless the CVE gets removed and re-added in a later scan).
If a new vulnerability feed version has new information about the cve, it won't be reflected in the previous scans.